geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/clearspring/CountMinSketch.scala

```diff
@@ -0,0 +1,204 @@
+/***********************************************************************
+ * Copyright (c) 2013-2017 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+/**
+  * Adapted from:
+  *
+  * stream-lib
+  * Copyright 2016 AddThis
+  *
+  * This product includes software developed by AddThis.
+  *
+  * This product also includes code adapted from:
+  *
+  * Apache Solr (http://lucene.apache.org/solr/)
+  * Copyright 2014 The Apache Software Foundation
+  *
+  * Apache Mahout (http://mahout.apache.org/)
+  * Copyright 2014 The Apache Software Foundation
```

Comment:
I think this requires a CQ.:(

Reply:
I created a CQ here: https://dev.eclipse.org/ipzilla/show_bug.cgi?id=14157
Used 'third party code request' but marked as modified and both source and binary